client/src/components/common/boost/BoostCalibration.tsx

```diff
@@ -53,6 +53,14 @@ export default function BoostCalibration({
     [handleSubmit],
   );
 
+  const displayCalibrationDistance = () => {
+    if (calibrationDiff) {
+      const calibratedDist = distTravelled + calibrationDiff;
+      return `${Math.round(calibratedDist * 100) / 100} m`;
```

Comment:
Could use [Number.prototype.toFixed](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed) as we want a rounded string representation.